asab/library/service.py

```
@@ -515,41 +523,56 @@ async def _read_favorites(self):
-				self.DisabledPaths = []
 			else:
-				self.Disabled = {}
-				self.DisabledPaths = []
-				for k, v in disabled_data.items():
-					if k.endswith('/'):
-						self.DisabledPaths.append((k, v))
-					else:
-						self.Disabled[k] = v
+				try:
+					disabled_data = yaml.load(disabled_file, Loader=yaml.CSafeLoader)
+				except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)